tools/post-install/src/main.rs

```diff
@@ -0,0 +1,361 @@
+use std::{
```

Comment:
Yeah exactly, I'm not a fan either. This is basically just a placeholder for the work @JamesGuthrie was doing in this direction

Reply:
If it's just intended to be a placeholder, can you remove it from this PR? It's not necessary for the packaging work.

Reply:
@JamesGuthrie It's serving a function, by "placeholder" I mean that we have other work in progress to handle versioned artifacts in a cleaner way, so it is intended that the `post-install` crate will be removed when that work is done, or at least when its ready to serve the same function. The reason it's worth having in the interim is to avoid packaging the extension without versioned artifacts.